docs/config.rst

```diff
@@ -152,6 +152,17 @@ Scrapyd includes an interface with a website to provide simple monitoring
 and access to the application's webresources.
 This setting must provide the root class of the twisted web resource.
 
+jobstorage
+-------
+
+A class that stores finished jobs. There are 2 implementations provided:
+
+* ``MemoryJobStorage`` (default) jobs are stored and memory and lost when the daemon is restarted
```

Comment:
default is actually:

scrapyd.jobstorage.MemoryJobStorage

let's put full names here:

scrapyd.jobstorage.MemoryJobStorage
and scrapyd.jobstorage.SqliteJobStorage